tg/controllers.py

```diff
@@ -671,7 +670,7 @@ def render(self) -> None:
     def _render(self) -> None:
         self._render_chats()
         self._render_msgs()
-        self._render_status()
+        # self._render_status()
```

Comment:
plz, remove whole line, instead of commenting it.

also, let's put status clean up here https://github.com/paul-nameless/tg/pull/136/files/147e991ca39864b127e2e638aabb666b694a943b#diff-d44e7f685a07442bfcb2ff6f977897a3R336